docs/website/docs/guides/deployment-configurations/index.md

```diff
@@ -37,13 +37,40 @@ When compiling programs, a list of target backends must be specified via
 | `cuda`         | NVIDIA GPU support via PTX for CUDA | `cuda` |
 | `rocm`         | **Experimental** <br> AMD GPU support via HSACO for ROCm | `rocm` |
 | `webgpu-wgsl`  | **Experimental** <br> GPU support on the Web via WGSL for WebGPU | `webgpu` |
-| `metal-spirv`  | **Stale - see [Issue#4370](https://github.com/openxla/iree/issues/4370)** <br> GPU support on Apple platforms via MSL for Metal | `metal` |
+| `metal-spirv`  | **Experimental** <br> GPU support on Apple platforms via MSL for Metal | `metal` |
```

Comment:
`metal` / `metal-spirv`, given both are recognized; also to be consistent with the following list.